README.md

```diff
@@ -34,14 +34,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is a way of marking up content that is meaningful - it is readable by the machine and search engines and useful for other coders as well.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+You can specify size for block - the height and width are changeable while the width of inline items are the width of its content.
+
+Inline items can be displayed side by side but block items create a new row/insert a line break - they can only be displayed one per row.
+
```

Comment:
Yes, another way to think about it is that block takes up the whole width while inline only takes up the space it needs.